docs/linux.md

```diff
@@ -78,10 +80,11 @@ Make sure to install ROCm v6
 
 ### Start Ollama
 
-Start Ollama using `systemd`:
+Start and Check Ollama service using `systemd`:
 
 ```bash
 sudo systemctl start ollama
+sudo systemctl status ollama
```

Comment:
I believe that would be nice to check if the ollama system service is running active here. @jmorganca how about adding status check here?